schema/crdb/dbinit.sql

```diff
@@ -2556,6 +2556,21 @@ INSERT INTO omicron.public.tuf_generation (
     (TRUE, 1)
 ON CONFLICT DO NOTHING;
 
+-- Trusted TUF root roles, used to verify TUF repo signatures
+CREATE TABLE IF NOT EXISTS omicron.public.tuf_trust_root (
+    -- Identity metadata (resource)
+    id UUID PRIMARY KEY,
+    time_created TIMESTAMPTZ NOT NULL,
+    time_modified TIMESTAMPTZ NOT NULL,
```

Comment:
I wonder if this should be an asset instead of a resource.  It seems like we don't really need `time_modified`.

Reply:
This is actually an asset, I need to fix the comment. We certainly don't need `time_modified`, but it's generated by the `Asset` derive. I can drop the derive but then we lose some of the nice trait methods for the ID field.